sdk/python/kubeflow/training/constants/constants.py

```diff
@@ -19,52 +19,56 @@
 APISERVER_TIMEOUT = 120
 
 # TFJob K8S constants
-TFJOB_GROUP = 'kubeflow.org'
-TFJOB_KIND = 'TFJob'
-TFJOB_PLURAL = 'tfjobs'
-TFJOB_VERSION = os.environ.get('TFJOB_VERSION', 'v1')
+TFJOB_GROUP = "kubeflow.org"
```

Comment:
Should we make this common as `KUBEFLOW_GROUP`.

Reply:
Nice, suggestion. I will change it.